pom.xml

```diff
@@ -349,17 +349,17 @@
       <dependency>
         <groupId>org.apache.httpcomponents</groupId>
         <artifactId>httpcore</artifactId>
-        <version>${httpcore.version}</version>
+        <version>4.4.13</version>
```

Comment:
Last one: need to update httpcore.version to 4.4.13 and change this to ${httpcore.version}